### DISCUSSION TITLE
When installing Neper software for testing, two commands failed. How should this be resolved?

### DISCUSSION THREAD

**Discussion by zlt2** [neperfepx/neper#752] [Q&A] [answered]
The following tests FAILED:
        211 - V/asymptote (Failed)
        326 - V/povray (Failed)
Errors while running CTest
Output from these tests are in: /mnt/d/a12/neper-main/src/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.

**Comment by rquey** [comment, marked as answer]
Did you install POV-Ray and Asymptote?  See [Installing Neper](https://neper.info/doc/introduction.html#installing-neper), and the tutorial [Installing Neper on Ubuntu 22](https://neper.info/doc/tutorials/install_ubuntu22.html#installing-neper-on-ubuntu-22) if you are on Ubuntu.
